README.md

```diff
@@ -193,6 +199,24 @@ This will analyze last 10k blocks and give you 25/50/75/99th/Max percentiles for
 
 `-tp 0.99` requests the 99th tip percentile across all the transaction in one block and calculates 25/50/75/99th/Max across all blocks
 
+### Bulk tracing
+You can trace multiple transactions at once using `seth trace` command. Example:
+```
+SETH_CONFIG_PATH=seth.toml go run cmd/seth/seth.go -n=Geth trace -f reverted_transactions.json
```

Comment:
I think we'll figure out some installation patterns for CLI soon. Should we just add is as `seth -n=Geth trace -f reverted_transactions.json` for simplicity?